sunkit_pyvista/plotter.py

```diff
@@ -140,3 +144,38 @@ def plot_solar_axis(self, length=2.5, arrow_kwargs={}, **kwargs):
                          scale='auto',
                          **defaults)
         self.plotter.add_mesh(arrow, **kwargs)
+
+    def plot_field_lines(self, gong_map, lon, lat, radius, nrho=25, rss=2.5, **kwargs):
+        """
+        Plots the field lines from `pfsspy`.
+
+        Parameters
+        ----------
+        gong_map : `pfsspy.map.GongSynopticMap`
+            to produce the field lines
+        lon : `astropy.units.quantity`
+        lat : `astropy.units.quantity`
+        radius : `float`
+        nrho : `int`
+        rss : `int`
```

Comment:
These all need descriptions.

Reply:
Yeah, these args are from pfsspy so I've left them as is for now, I'll understand them better and write their descriptions